### ISSUE TITLE
No simple: `@rendition` value for indent?

### ISSUE THREAD

**Issue by martindholmes** [TEIC/TEI-Simple#30]
Indented first lines in paras and other blocks are common in all early modern texts. The current simplePrint spec doesn't provide anything for this. I propose simple:indentFirst, or failing that, from CSS, simple:textIndent.

**Comment by martinascholger** [comment]
+1 I have a slight preference for the second (CSS-alike) variant. I have noticed that we have not used the camelcase spelling for the xml:id values so far. Should we stick to small letters (e.g. simple:textindent)?

**Comment by martindholmes** [comment]
Yes, I'd go with simple:textindent. It can be derived from the CSS property in an automated way (although the reverse is harder).

**Comment by Conal-Tuohy** [comment]
The name of the related CSS property is actually [`text-indent`](https://www.w3.org/wiki/CSS/Properties/text-indent) (with a hyphen). I'd suggest just using the same name.

**Comment by martindholmes** [comment]
We have no other hyphenated simple: properties, so I didn't want to introduce one. The convention seems to be to remove the hyphen:

CSS: small-caps
simple:smallcaps
